modules/instrument_manager/jsx/uploadForm.js

```diff
@@ -52,7 +52,7 @@ class InstrumentUploadForm extends Component {
       }
       if (data.error) {
          swal({
```

Comment:
```suggestion
         swal.fire({
```

and append `import swal from 'sweetalert2';` at the top of this file

Reply:
Thanks!

Reply:
It looks like the import line is already there.

Reply:
i don't think the import line is there yet. you might be thinking of the other uploadForm.js in the doc repo module. thinking of it, does the instrument_manager uploadForm jsx file need to be in this PR at all? there are also 2 other swal calls that need to be updated to sweetalert2 if adding changes to this file is intentional

Reply:
Oh oops I misread this. This file shouldn't be committed at all - my mistake. I'll remove it.